apis/flowcollector/v1beta1/flowcollector_types.go

```diff
@@ -157,9 +157,9 @@ const (
 	FlowRTT     AgentFeature = "FlowRTT"
 )
 
-// Name of a ebpf agent alert.
+// Name of an eBPF agent alert.
 // Possible values are:<br>
-// - `NetObservDroppedFlows`, which is triggered when eBPF agent hashmap table is full.<br>
+// `NetObservDroppedFlows`, which is triggered when the eBPF agent is dropping flows, such as when the BPF hashmap is full.<br>
```

Comment:
can we state the other possible reason for drop here i.e limiter capacity exceeded and possible recovery config if any ? from what I see I dn't think there is anything can be done to avoid limiter more of internal go limits ?

Reply:
done

Reply:
the capacity limiter is not related to go limits (if you're talking about the GOMEMLIMIT), it's more some sort of backpressure management, cf the warning text that is logged when it's triggered: https://github.com/netobserv/netobserv-ebpf-agent/blob/bf91cbef0008f6d6bc8b1c748729c18ec6b14d35/pkg/flow/limiter.go#L50-L53

Reply:
Cool Thanks!!

Reply:
cc'd @skrthomas this might need to be doc under the ebpf agent alerting WDYT?